app/src/main/java/com/example/cupcake/ui/StartOrderScreen.kt

```diff
@@ -49,6 +49,7 @@ import com.example.cupcake.ui.theme.CupcakeTheme
 @Composable
 fun StartOrderScreen(
     quantityOptions: List<Pair<Int, Int>>,
+    onNextButtonClicked:(Int)-> Unit,
```

Comment:
```suggestion
    onNextButtonClicked:(optionValue: Int)-> Unit,
```
Here we can name the lambda argument